src/web/layouts/DecideLayout.tsx

```diff
@@ -41,30 +43,28 @@ export const DecideLayout = ({ CAPI, NAV }: Props): JSX.Element => {
 	};
 	const palette = decidePalette(format);
 
+	// TODO we can probably better express this as data
 	switch (display) {
 		case Display.Immersive: {
-			switch (design) {
-				case Design.Comment:
-				case Design.Editorial:
-				case Design.Letter:
-					return (
-						<ImmersiveLayout
-							CAPI={CAPI}
-							NAV={NAV}
-							palette={palette}
-							format={format}
-						/>
-					);
-				default:
-					return (
-						<ImmersiveLayout
-							CAPI={CAPI}
-							NAV={NAV}
-							palette={palette}
-							format={format}
-						/>
```

Comment:
great catch didn't notice they both returned the same 👍

Reply:
Ha yes was a bit confused by this!